content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -736,15 +736,25 @@ for more information.
 
 ### SecurityContextDeny {#securitycontextdeny}
 
-{{< feature-state for_k8s_version="v1.0" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="deprecated" >}}
 
 {{< caution >}}
-This admission controller plugin is **outdated** and **incomplete**, it may be
-unusable or not do what you would expect. It was originally designed to prevent
-the use of some, but not all, security-sensitive fields. Indeed, fields like
-`privileged`, were not filtered at creation and the plugin was not updated with
-the most recent fields, and new APIs like the `ephemeralContainers` field for a
-Pod.
+The Kubernetes project recommends that you **do not use** the
+`SecurityContextDeny` admission controller.
+
+This admission controller plugin is deprecated and disabled by default. If you
+wish to enable it, you must specifically enable the`SecurityContextDeny` feature
+gate as well as enabling this admission controller.
```

Comment:
If you choose to enable the `SecurityContextDeny` admission controller, you must enable the `SecurityContextDeny` feature gate ...

Reply:
I took into account your comment just avoiding the repetition.